src/ica_api.c

```diff
@@ -2504,10 +2504,18 @@ unsigned int ica_3des_ctr(const unsigned char *in_data, unsigned char *out_data,
 			 unsigned int direction)
 {
 #ifdef ICA_FIPS
+	/* Note that the FIPS ctr check cannot detect ctr wraps
```

Comment:
why is the check not added to ica_des_ctr altough the commit msg says so ?

Reply:
Because DES is blocked in FIPS mode anyway. But good catch on the commit msg! I will change it.

Reply:
> DES is blocked in FIPS mode anyway

Yeah, right didnt think of that. So its just the commit msg